tests/saitests/sai_qos_tests.py

```diff
@@ -2140,9 +2146,22 @@ def runTest(self):
                     pkts_num -= diff
                     expected_wm -= diff * cell_occupancy
                     fragment = total_shared - expected_wm
+
+                pkts_total += pkts_num
+                if 'cisco-8000' in asic_type:
+                    # send full queue fill level
+                    pkts_num = pkts_total
```

Comment:
Combine line 2153 and 2156 to make it more concise.

Reply:
done